packages/calypso-e2e/package.json

```diff
@@ -20,7 +20,7 @@
 	"dependencies": {
 		"@types/totp-generator": "^0.0.3",
 		"mailosaur": "^7.3.1",
-		"playwright": "^1.19.2",
+		"playwright": "1.22.2",
```

Comment:
Optional question -- but have we formally decided how we want to handle version bumps with Playwright? I see here we're going from the caret to an explicit version. I think that's fine, especially if we want to be very explicit about what version we're upgrading to in the future!

But, if from time to time we will occasionally just want to take whatever is the newest, we can probably keep the caret.

The caret just means that when you run `yarn upgrade`, it will take the most recent non-major version.

Reply:
Sorry, this is because I always don't think and type `yarn up playwrighrt@version`. 

It would be nice to have the caret thing to be the default in yarn settings, but I don't know if that's possible.